internal/handlers/tigris/msg_listdatabases.go

```diff
@@ -17,10 +17,94 @@ package tigris
 import (
 	"context"
 
+	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgListDatabases implements HandlerInterface.
 func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	return nil, errNotImplemented
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	var filter *types.Document
+	if filter, err = common.GetOptionalParam(document, "filter", filter); err != nil {
+		return nil, err
+	}
+
+	common.Ignored(document, h.L, "comment", "authorizedDatabases")
+
+	databaseNames, err := h.driver.ListDatabases(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO https://github.com/FerretDB/FerretDB/issues/591
+	nameOnly, _ := common.GetOptionalParam(document, "nameOnly", false)
+
+	databases := types.MakeArray(len(databaseNames))
+	for _, databaseName := range databaseNames {
+		res, err := h.driver.DescribeDatabase(ctx, databaseName)
+		if err != nil {
+			return nil, lazyerrors.Error(err)
+		}
+
+		// iterate over result to collect sizes
```

Comment:
Very obvious comment.  What did you mean?

Reply:
https://github.com/FerretDB/FerretDB/blob/9e28a0add5a70150d8ce6789478a9c370fb3f8a9/internal/handlers/pg/msg_listdatabases.go#L56

Reply:
> https://github.com/FerretDB/FerretDB/blob/9e28a0add5a70150d8ce6789478a9c370fb3f8a9/internal/handlers/pg/msg_listdatabases.go#L56

I would suggest removing this comment. For consistency, it could be removed there also.